main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        cloud
+//  action 1:    drops water
+cloud.dropWater();
+//  action 2:    disappears
+cloud.disappear();
+//  action 3:    turns violet
+cloud.turnViolet();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        water
+//  action 1:    freezes
+water.freeze();
+//  action 2:    evaporates
+water.evaporate();
+//  action 3:    boils
+water.boil();
+//  3)            
+//  Noun:        car
+//  action 1:    is standing still
+car.stand();
+//  action 2:    is driving
+car.drive();
+//  action 3:    turns left
+car.turnLeft():
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        Compressor
+//  action 1:    squishes audio
+compressor.squish();
+//  action 2:    does not squish audio
+compressor.noSquish();
+//  action 3:    shows overload
+compressor.overload();

```

Comment:
`does not squish audio` is not an action.  it is a description about the state of the compressor.
- [ ] pick a different action, that contains a verb.

`overload` is not an action.   `show overload` is an action.  you're missing the verb here.
- [ ] add the verb
- [ ] revise